### PR TITLE
Fix room user count is always 1

### PR DESCRIPTION
**Pull request by bxtsang**

### CODE REVIEW

File: webproject/src/main/java/com/google/sps/data/Room.java [hunk 1]
```diff
@@ -54,23 +53,25 @@ private Room (Builder builder) {
...
 
+    public void addUser(String userEmail, String roomId) {
+        UserRoom userRoom = new UserRoom(userEmail, roomId);
+        userRoom.save();
+    }
+
     public void save() throws IOException {
         String roomJson = gson.toJson(this);
-        Firebase.sendRequest("https://summer20-sps-47.firebaseio.com/rooms.json", "POST", roomJson);
+        String firebaseResponse = Firebase.sendRequest("https://summer20-sps-47.firebaseio.com/rooms.json", "POST", roomJson);
```
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/data/Room.java:70]: should use the common firebase util

File: webproject/src/main/java/com/google/sps/data/Room.java [hunk 2]
```diff
@@ -54,23 +53,25 @@ private Room (Builder builder) {
...
+        Map<String, String> firebaseData = gson.fromJson(firebaseResponse, Map.class);
+
+        this.addUser(this.creator, firebaseData.get("name"));
```
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/data/Room.java]: is name the unique identifier of a user?

File: webproject/src/main/webapp/js/room-my-order.js [hunk 3]
```diff
@@ -7,12 +7,16 @@ window.onload = function() {
 }
 
 async function getMyOrder() {
-    let orderResponse = await fetch(`/myOrder?roomId=${roomID}`);
-    let myOrders = await orderResponse.json();
-    let myOrderItems = Object.values(myOrders);
+    // let orderResponse = await fetch(`/myOrder?roomId=${roomID}`);
+    // let myOrders = await orderResponse.json();
```
Comment (kevinluvian) [inline webproject/src/main/webapp/js/room-my-order.js]: remove dead code

**Review by kevinluvian — CHANGES_REQUESTED** [review]

File: webproject/src/main/java/com/google/sps/data/Room.java (hunk 2 above)
Comment (bxtsang) [reply webproject/src/main/java/com/google/sps/data/Room.java]: name is the key for the ID from firebase's response!

**Comment by bxtsang** [comment]
fixed in a separate PR

**Pull request closed without merging**